src/App.css

```diff
@@ -1,3 +1,8 @@
+body{
+  background-image: url(../src/images/background3.jpg);
+  background-size: cover;
```

Comment:
Would this background repeat? would it be centered as well?

I would like to see how this looks? Is the image big enough such that it doesn't stretch and get rastered?